src/db/api.js

```diff
@@ -0,0 +1,15 @@
+const SchoolType = require("./models");
+
+const createSchoolType = () => {
+  SchoolType.sync({ force: true }).then(() => {
+    return SchoolType.create({
+      schoolType: "szkoła podstawowa"
```

Comment:
I think we could go with enum here, so no magic strings ;)

Reply:
Yes. It need argument, but as a POC I insert that wired string.